routes/tasks.js

```diff
@@ -6,16 +6,38 @@ const { createTask, updateTask, updateSelfTask, getTasksValidator } = require(".
 const authorizeRoles = require("../middlewares/authorizeRoles");
 const { APPOWNER, SUPERUSER } = require("../constants/roles");
 const assignTask = require("../middlewares/assignTask");
-const cache = require("../utils/cache");
+const { cache, invalidateCache } = require("../utils/cache");
+const { TASKS_ALL } = require("../constants/cacheKeys");
 
-router.get("/", getTasksValidator, cache(), tasks.fetchTasks);
+router.get("/", getTasksValidator, cache({ invalidationKey: TASKS_ALL, expiry: 10 }), tasks.fetchTasks);
```

Comment:
can we rename cache to a more descriptive middleware such as `getCachedResponse ` or something on similar lines